community/kernel/src/main/java/org/neo4j/kernel/impl/api/KernelTransactionImplementation.java

```diff
@@ -108,7 +109,7 @@
  * as
  * {@code TransitionalTxManagementKernelTransaction} is gone from {@code server}.
  */
-public class KernelTransactionImplementation implements KernelTransaction, TxStateHolder
+public class KernelTransactionImplementation implements KernelTransaction, TxStateHolder, ExecutionStatistics
```

Comment:
field statementOperations is not used now please remove it

Reply:
I removed the field [here](https://github.com/neo4j/neo4j/pull/11455) instead, is that OK. Saves me some rebasing.

Reply:
sure